website/index.html

```diff
@@ -0,0 +1,294 @@
+                <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z" clip-rule="evenodd" />
+              </svg>
+              Improved decidability
+            </h3>
+            <p>Fe limits dynamic program behavior to improve decidability and allow more precise gas cost estimation.</p>
+          </div>
+          <div class="p-6 bg-gray-100 shadow-2xl">
+            <h3 class="text-xl font-semibold mb-2">
+              <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 inline-block align-text-top mr-1 text-green-600" viewBox="0 0 20 20" fill="currentColor">
+                <path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm3.707-9.293a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z" clip-rule="evenodd" />
+              </svg>
+              Standard library
+            </h3>
+            <p>Fe aspires to offer a rich standard library to assist with common tasks of smart contract development.</p>
+          </div>
+        </div>
+        <div class="lg:max-w-2xl lg:flex-1 lg:-mr-18">
+          <div class="relative overflow-hidden md:rounded-xl md:rounded-b-none xl:rounded-b-xl shadow-2xl flex bg-gray-500 pb-6 md:pb-0">
+            <div class="absolute inset-0 bg-black bg-opacity-75"></div>
+            <div class="relative w-full flex flex-col">
+              <div class="flex-none h-11 flex items-center px-4">
+                <div class="flex space-x-1.5">
+                  <div class="w-3 h-3 border-2 rounded-full bg-red-500 border-red-500">
+                  </div>
+                  <div class="w-3 h-3 border-2 rounded-full bg-yellow-300 border-yellow-300">
+                  </div>
+                  <div class="w-3 h-3 border-2 rounded-full bg-green-400 border-green-400">
+                  </div>
+                </div>
+              </div>
+              <div class="relative border-t border-white border-opacity-10 min-h-0 flex-auto flex flex-col">
+                <div class="hidden md:block absolute inset-y-0 left-0 bg-black bg-opacity-25" style="width: 50px;">
+                </div>
+                <div class="w-full flex-auto flex min-h-0 overflow-auto">
+                  <div class="w-full relative flex-auto">
+                    <pre class="flex min-h-full text-sm md:text-sm">
+                      <div aria-hidden="true" class="hidden md:block text-white text-opacity-50 flex-none py-4 pr-4 text-right select-none" style="width: 50px;">1
+  2
+  3
+  4
+  5
+  6
+  7
+  8
+  9
+  10
+  11
+  12
+  13
+  14
+  15
+  16
+  17
+  18
+  19
+  20
+  21
+  22
+  23
+
+                      </div>
+                      <code class="flex-auto relative block text-white pt-4 pb-4 px-4 overflow-auto">contract Ownable:
```

Comment:
I think we should use guest book demo here, since ownable is used as an inherited.

Reply:
Ah good point. Eventually it would be nice to demo something that uses the new `string<100>` syntax when it is ready. I'll use the guest book for now.

Reply:
I've put in some comments and syntax highlighting (Python for now) because the example is shorter and I needed it to consume a bit more space to fit in nicely

![image](https://user-images.githubusercontent.com/521109/117708583-47bfd400-b1d0-11eb-877f-ae82e8e526b5.png)